routes/index.js

```diff
@@ -1,13 +1,17 @@
 var express = require("express");
 var router = express.Router();
+const DocumentService = require("../services/document/document.service");
 
 /* GET home page. */
 router.get("/", async function(req, res, next) {
+  const documents = await DocumentService.getAllDocuments();
```

Comment:
what does that do ?
All documents of a specific key user ?

Reply:
yes